app/models/cas_session.rb

```diff
@@ -0,0 +1,6 @@
+class CasSession
```

Comment:
Could this be named MongoCasSession or something like that, just in case we want to add more of this sort of thing later?  (It could still be storing a document called "cas_sessions" in MongoDB; I mostly am interested in making room for non-Mongo implementations.)